src/libraries/Common/tests/System/Xml/XPath/FuncExpressions/BasicsTests.cs

```diff
@@ -100,5 +101,17 @@ public static void BasicsTest164()
             Utils.XPathNodesetTestThrows<System.Xml.XPath.XPathException>(xml, testExpression,
                 startingNodePath: startingNodePath);
         }
+
+        [Fact]
+        public static void XPathDocumentNullStream()
+        {
+            Assert.Throws<ArgumentNullException>(() => new XPathDocument(default(Stream)));
```

Comment:
why not `null`?

Reply:
there is also TextReader overload and we want to make sure it picks the right one

Reply:
Apples vs oranges. No strong preference either way. More important to test behavior rather than if we do it one way or another